### PR TITLE
Update group-update.md

### PR DESCRIPTION
**Pull request by yyuank**

### CODE REVIEW

File: api-reference/beta/api/group-update.md [hunk 1]
```diff
@@ -90,12 +91,58 @@ Content-length: 211
...
   "displayName": "displayName-value",
-  "groupTypes": [
-    "groupTypes-value"
-  ],
-  "mail": "mail-value",
-  "mailEnabled": true,
-  "mailNickname": "mailNickname-value"
+}
+```
+# [C#](#tab/csharp)
```
Comment (Lauragra) [inline api-reference/beta/api/group-update.md:96]: @yyuank , did you add these snippets? Generally they are added via automation when files are merged into master.

Comment (yyuank) [reply api-reference/beta/api/group-update.md:96]: No, I didn't add those snippets. I copied everything from the first example and modified it for the second one. I saw those snippets are from the first example, so they were already there when I started to edit the file.

**Comment by yyuank** [comment]
@Lauragra thank you for merging the file edits. I don't see C#, JavaScript examples in the requests in the doc. Is this expected?

**Comment by Lauragra** [comment]
> @Lauragra thank you for merging the file edits. I don't see C#, JavaScript examples in the requests in the doc. Is this expected?

Yes, since you updated the request example, we need to regenerate the associated snippets. That will happen in the weekly snippet automation that happens on Tuesdays, so they'll be added next Tuesday.

**Pull request merged**